module/Company/src/Company/Service/Company.php

```diff
@@ -482,9 +482,6 @@ public function saveJobData($languageNeutralId, $jobs, $data, $files)
         $id = -1;
 
         foreach ($jobs as $lang => $job) {
-            if ($job->getActive() !== '1') {
-                continue;
-            }
```

Comment:
Did you check that you can also save a job without English or Dutch translation?